src/Unsafe/Coerce.purs

```diff
@@ -8,20 +8,13 @@ module Unsafe.Coerce
 -- | (that is, the caller's) responsibility to ensure that the underlying
 -- | representation for both types is the same.
 -- |
--- | One application for this function is to avoid doing work that you know is a
--- | no-op because of newtypes. For example, if you have an `Array (Conj a)` and you
--- | want an `Array (Disj a)`, you could do `Data.Array.map (runConj >>> Disj)`, but
--- | this performs an unnecessary traversal. `unsafeCoerce` accomplishes the same
--- | for free.
+-- | Note: because this function is extraordinarily flexible, type inference
+-- | can greatly suffer. Whenever you use it, you should add type annotations
+-- | to clarify what the type of it's input value is and the type of it's
+-- | output value.
 -- |
--- | It is highly recommended to define specializations of this function rather than
```

Comment:
I would like to keep this part in, as I think it's a bit more clear. As for the example, how about:

```
fromBoolean :: Boolean -> Json
fromBoolean = unsafeCoerce
```

Reply:
Fixed in latest commits.